src/main/java/org/cyclops/integrateddynamics/part/aspect/Aspects.java

```diff
@@ -122,7 +129,8 @@ public static void load() {}
                             // Catch possible errors
                         }
                         return null;
-                    }).handle(AspectReadBuilders.PROP_GET_NBT, "tile").buildRead();
+                    }).withUpdateType(AspectUpdateType.BLOCK_UPDATE)
```

Comment:
Doesn't NBT frequently change even without a block update?

Reply:
True, this was part of a small test I did, but should be removed indeed.